### ISSUE TITLE
Loading Json Registry Encoding Error 

### ISSUE THREAD

**Issue by MSchwesig** [mdomke/schwifty#76]
Hello first of all great module. 

However, when loading the module I get a chamap error in Windows. 

```
Exception: UnicodeDecodeError
'charmap' codec can't decode byte 0x8d in position 416: character maps to <undefined>.
```

The problem is loading the JSON files. 
Since all files are always in UTF-8 you can just specify the format when loading, then there will be no problem with the different operating systems. 

Simply change  line 47 in the **regitry.py**  from
` with entry.open() as fp:`

to 
`   with entry.open(encoding='utf-8') as fp:`

this should also solve the Docker problem:
https://schwifty.readthedocs.io/en/latest/troubleshooting.html

Best Regards

Max Schwesig

**Comment by ckoehn** [comment]
See #75

**Comment by MSchwesig** [comment]
Ah sorry didn't see that one.

**Comment by mdomke** [comment]
This has been fixed in release `2022.04.0`.
